### PR TITLE
5G PoC - Speed Test Tiers

### PR DESCRIPTION
**Pull request by samgutentag**
updated Speed Test Tiers documentation, for more info, see https://discord.com/channels/404106811252408320/730418759298318346/1029483894023671870

### CODE REVIEW

File: docs/5g-on-helium/mobile-poc.mdx [hunk 1]
```diff
@@ -129,16 +127,88 @@ must:
...
+:::
+
+**Tiered Speed Test values introduce a new multiplier to the Genesis Rewards calculations:**
+
 A 5G Hotspot will automatically perform a Speed Test approximately every 12 hours with the results
 being averaged by the MOBILE PoC Oracle over the previous `6` tests, depending on the availability
 of the data. This moving average is reset following a speed test gap of `>12 hours` and a minimum of
 `2` Speed Test results are required to be eligible for Genesis Rewards.
 
+- Speed Test Results are put into Tiers based on the minumum Tier of each Download, Upload and
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: minumum Tier > minimum value

File: docs/5g-on-helium/mobile-poc.mdx [hunk 2]
```diff
@@ -129,16 +127,88 @@ must:
...
+  Latency. (logical `AND`)
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Latency. (logical `AND`) > Latency (logical `AND`).

File: docs/5g-on-helium/mobile-poc.mdx [hunk 3]
```diff
@@ -129,16 +127,88 @@ must:
...
+- Speed Test results that do not meet the minimum requirements for any Download, Upload or Latency
+  are considered to have `Failed` and are not eligible for Genesis Rewards until the Speed Test
+  Average is improved. (logical `OR`)
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: same as above

File: docs/5g-on-helium/mobile-poc.mdx [hunk 4]
```diff
@@ -14,52 +14,44 @@ that Hotspots are honestly representing their location and the wireless network
...
 tolerance for downtime in infrastructure that is beyond the 5G Hotspot Owner's control.
 
-To qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio must:
+To qualify for Genesis Rewards in a Reward Period, a CBRS Small Cell Radio must:
 
 - Be connected to a 5G Hotspot.
 - Be properly installed.
   - Be registered with the SAS (Outdoor Radios Only).
 - Broadcast at least 1 "heartbeat" per Reward Period.
+- Meet Speed Test Metrics of at least `Poor` on all metrics.
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Speed Tests are not launched yet, not an immediate requirment.

File: docs/5g-on-helium/mobile-poc.mdx [hunk 5]
```diff
@@ -14,52 +14,44 @@ that Hotspots are honestly representing their location and the wireless network
...
-  - _Genesis Phase 2_:
-    [Time Period Triggered Rewards with Minimum Delay](/5g-on-helium/mobile-poc#genesis-phase-2-time-period-triggered-rewards-with-minimum-delay)
-  - _Genesis Phase 3_:
-    [Speed Test and Quality of Service](/5g-on-helium/mobile-poc#genesis-phase-3-speed-tests-and-quality-of-service)
-  - _Genesis Phase 4_:
-    [Heartbeats and Consistency of Service](/5g-on-helium/mobile-poc#genesis-phase-4-heartbeats-and-reliability-of-service)
-- **Stage 2: ** 5G Mappers Proof of Coverage
-- **Stage 3: ** Additional Methods of Proof of Coverage
+  - _Current Implementation_:
+    [Speed Test and Quality of Service](/5g-on-helium/mobile-poc#speed-tests-and-quality-of-service)
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Same as below, this is not current yet.

File: docs/5g-on-helium/mobile-poc.mdx [hunk 6]
```diff
@@ -74,82 +66,258 @@ brief interruptions.
...
 ---
 
-## Stage 1: Genesis Rewards
+## Speed Tests and Quality of Service
 
-:::info Reward Targets
+> - The reward Period is approximately 24 hours, with Reward submission occurring at approximately
+>   01:30 UTC.
+> - 5G Hotspots perform periodic Speed Tests which will be considered in Reward calculations.
+> - A minimum Download speed, minimum Upload speed, and maximum Latency will be required by each 5G
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: If section marked as "current" it should not mention future plans.

File: docs/5g-on-helium/mobile-poc.mdx [hunk 7]
```diff
@@ -14,52 +14,44 @@ that Hotspots are honestly representing their location and the wireless network
...
+    [Speed Test and Quality of Service](/5g-on-helium/mobile-poc#speed-tests-and-quality-of-service)
+  - _Work in Progress_:
+    [Heartbeats and Consistency of Service](/5g-on-helium/mobile-poc#heartbeats-and-reliability-of-service)
+- **Stage 2: ** [5G Mappers Proof of Coverage](/5g-on-helium/mobile-poc#mobile-mappers)
+- **Stage 3: ** [Additional Methods of Proof of Coverage](/5g-on-helium/mobile-poc#future-roadmap)
 
 ---
 
-## Current MOBILE PoC Reward Elegibility
+## PoC Reward Elegibility
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Not sure why this section is needed. It currently says that speed tests are required, and next section says that Speed Tests will be required.

File: docs/5g-on-helium/mining-mobile.mdx [hunk 8]
```diff
@@ -132,11 +132,18 @@ All properly installed and registered 5G Hotspots will earn rewards for particip
...
 
-| Small Cell Radio Type | Reward Weight |
-| :-------------------- | :-----------: |
-| High-Power Outdoor\*  |     4.0x      |
-| Outdoor               |     2.5x      |
-| Indoor                |     1.0x      |
+:::note
+
+Begining in [Genesis Phase 3](./mobile-poc#genesis-phase-3-speed-tests-and-quality-of-service) 5G
+Hotspots perform Speed Tests that impact Rewards Scaling Multipliers in additoin to Small Cell Type.
```
Comment (rawrmaan) [inline docs/5g-on-helium/mining-mobile.mdx]: Typo: `additoin`

File: docs/5g-on-helium/mobile-poc.mdx [hunk 9]
```diff
@@ -74,82 +64,260 @@ brief interruptions.
...
 
-### Genesis Phase 2: Time Period Triggered Rewards with Minimum Delay
+### Tiered Speed Test
 
-:::info
+In response to community feedback and further exploration by Chris from DeWiPulse for his
+[analysis of and commentary on public Internet data in the United States](https://dewipulse.com/connectivity-where-it-matters/)
+Speed Test results are categorized into one of four Tiers.
+
+| Speed Test Tier | Reward Multiplier | Requirement (speeds in Mbps)                                   |
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Optionally for consistency how the values are displayed in the table:
Requirement (speeds in Mbps) > Requirement (speeds in Mbps, latency in ms) and remove ms from table

File: docs/5g-on-helium/mobile-poc.mdx [hunk 10]
```diff
@@ -74,82 +64,260 @@ brief interruptions.
...
 
 :::
 
-> - Reward Period is approximately 1440 blocks and closely aligns with 00:00 UTC to 23:59 UTC.
-> - Reward submission occurs approx 15 minutes after the Reward Period ends, around 00:00 UTC.
+:::warning **backhaul requirements will evolve**
 
-At this Milestone, to qualify for the Genesis Rewards in a Reward Period, a CBRS Small Cell Radio
-must:
+As the needs and realities of the 5G Hotspot network change, internet backhaul requirements will
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: internet > Internet ?

File: docs/5g-on-helium/mobile-poc.mdx [hunk 11]
```diff
@@ -74,82 +64,260 @@ brief interruptions.
...
+
+```bash
+Download    200 ->  `Acceptable`
+Upload       40 ->  `Acceptable`
+Latency     150 ->  `Fail`
+---------------------------------
+Speed Test Tier ->  `Fail`        -> 0.0x Multiplier
+```
+
+### Statistics on Explorer
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Should the be an article "the" before Explorer?

File: docs/5g-on-helium/mobile-poc.mdx [hunk 12]
```diff
@@ -74,82 +64,260 @@ brief interruptions.
...
+
+Speed Test metrics can be viewed on the 5G Hotspot's page on Explorer under the `5G Statistics` tab.
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: They cannot be viewed yet in the way they are described in this doc.

File: docs/5g-on-helium/mobile-poc.mdx [hunk 13]
```diff
@@ -74,82 +64,260 @@ brief interruptions.
...
+
+Here are some examples of each status and a brief description of each.
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Also below screens are for derived data. First iteration is to update the Explorer to displat info only screens, you can see them here https://github.com/helium/explorer/issues/1059 and in figma.

File: docs/5g-on-helium/mobile-poc.mdx [hunk 14]
```diff
@@ -74,82 +64,260 @@ brief interruptions.
...
+    <div class="col col--6" style={{ marginBottom: 0 }}>
+      <img
+        className="explorerSidebar"
+        src={useBaseUrl('img/5g-on-helium/explorer-5g-stats-no-recent.png')}
+      />
+    </div>
+    <div class="col col--6" style={{ marginBottom: 0 }}>
+      <b>No Recent Speed Test Data</b>
+      <p>
+        This status appears when a 5G Hotspot could not be performed in the previous 24 hours. For
```
Comment (meowshka) [inline docs/5g-on-helium/mobile-poc.mdx]: Looks like a few words are missing in the sentence.

File: docs/5g-on-helium/mobile-poc.mdx (hunk 11 above)
Comment (samgutentag) [reply docs/5g-on-helium/mobile-poc.mdx]: swapping for "Statistics on Helium Explorer" to match other docs references

File: docs/5g-on-helium/mobile-poc.mdx (hunk 12 above)
Comment (samgutentag) [reply docs/5g-on-helium/mobile-poc.mdx]: replaced with "Soon, Speed Test metrics _will_ be viewable.

**Review by meowshka — DISMISSED** [review]

**Review by meowshka — APPROVED** [review]

**Pull request merged**